src/wp-includes/rest-api.php

```diff
@@ -1393,6 +1393,12 @@ function rest_validate_value_from_schema( $value, $args, $param = '' ) {
 					return new WP_Error( 'rest_invalid_param', sprintf( __( '%s is not a valid IP address.' ), $param ) );
 				}
 				break;
+			case 'uuid':
+				if ( ! wp_is_uuid( $value ) ) {
+					/* translators: %s is an invalid universally unique identifier (UUID). */
```

Comment:
This isn't exactly right, `$param` is the name of the parameter. So it would read `someuuid is not a valid UUID` for instance.

Reply:
Ah, you're right 👍